src/GeositeFramework/js/Plugin.js

```diff
@@ -443,6 +462,17 @@ require(['use!Geosite',
             return view;
         }
 
+        function toggleTabletSidebarPluginText(visible) {
```

Comment:
`toggle` implies that you can invert the current setting.  If this really requires a bool arg, I'd rename it to `setSideBarPluginTextVisibility`, and drop the `Tablet` since that really hasn't got anything to do with the function implementation.